src/pixelator/pixeldataset/precomputed_layouts.py

```diff
@@ -232,15 +232,20 @@ def data():
         for sample_name, layout in precomputed_layouts:
             if layout is None:
                 continue
+            if layout.is_empty:
+                continue
             layout_with_name = layout.lazy.with_columns(
                 sample=pl.lit(sample_name)
             ).pipe(zero_fill_missing_markers, all_markers=all_markers)
             yield layout_with_name
 
-    return PreComputedLayouts(
-        pl.concat(data(), rechunk=False),
-        partitioning=["sample"] + PreComputedLayouts.DEFAULT_PARTITIONING,
-    )
+    try:
+        return PreComputedLayouts(
+            pl.concat(data(), rechunk=False),
+            partitioning=["sample"] + PreComputedLayouts.DEFAULT_PARTITIONING,
+        )
+    except ValueError:
```

Comment:
Can this be included as one of these pixelator exceptions? What does it mean `ValueError` here?

Reply:
In this case the `ValueError` is raised by `pl.concat` if there is no data in `data()`, and since we catch it to make sure we return an empty precomputed layout object I don't think we need to convert it into a specific handled exception.